container/webui/Dockerfile

```diff
@@ -33,6 +34,7 @@ RUN zypper ar -p 95 -f http://download.opensuse.org/repositories/devel:openQA/15
 
 COPY openqa-ssl.conf openqa.conf /etc/apache2/vhosts.d/
 COPY run_openqa.sh /root/
+RUN sed -i "s/#ServerName your.server.name/ServerName $HOST/" /etc/apache2/vhosts.d/openqa.conf
```

Comment:
This is already done by script/configure-web-proxy which seems to be not used yet here which is also why the above "a2enmod" calls are needed. Can you reuse configure-web-proxy?